addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -746,11 +738,10 @@ registerModel({
             if (rtcSession) {
                 rtcSession.reset();
             }
-            const dataChannel = this._dataChannels[token];
-            if (dataChannel) {
-                dataChannel.close();
+            const rtcDataChannel = this.messaging.models['RtcDataChannels'].findFromIdentifyingData({ token });
+            if (rtcDataChannel) {
+                rtcDataChannel.delete();
```

Comment:
Not for this PR, but when we have all rtc models, it looks like data channel should be more closely related to peer (probably with causality).